### ISSUE TITLE
UI improvement: whole row should be clickable to get to the detail of a block

### ISSUE THREAD

**Issue by konopkja** [ethproofs/ethproofs#233]
### Is your feature request related to a problem? Please describe.

Accessibility of getting to the detail page is limited to a small arrow icon on the right side of the table.

### Describe the solution you'd like

For easier accessibility, the whole highlighted area should be clickable to get to the detail page.

![Image](https://github.com/user-attachments/assets/44870c04-b499-46dd-b1d6-b0d43861ce04)

### Describe alternatives you've considered

NA

**Comment by nloureiro** [comment]
With the new design, we need to validate whether this is still valid


Closing this issue for now and circling back on the new design user experience
